### ISSUE TITLE
EDC-management: "Bad Request" when creating a "policy"

### ISSUE THREAD

**Issue by edwo-dev** [eclipse-edc/DataDashboard#65]
# Bug Report

## Describe the Bug

Policy can not be created.
'Bad Request' error is thrown.

### Expected Behavior

Policy is created.

### Observed Behavior

1. Throws error message
2. Does not create policy

## Steps to Reproduce

1. Go to 'Policies'
2. Click on 'Create policy'
3. Insert `id`
4. Click 'Save'
5. See error

## Context Information

- Used version
  - [DataDashboard https://github.com/eclipse-edc/DataDashboard/commit/550f6d0b557bf76b5ab7cf665d6c1a98cb156186]
  - [EDC v0.2.0]
- OS: [Windows/CentOS7]

## Error Information

`zone.js:2680     POST http://localhost:9192/api/management/v2/policydefinitions 400 (Bad Request)`

## Detailed Description

Request payload:
```
{
    "policy": {
        "@type": "SET"
    },
    "id": "test"
}
```

Response:
```
{
  "servlet":"EDC-management",
  "message":"Bad Request",
  "url":"/api/management/v2/policydefinitions",
  "status":"400"
}
```

**Comment by ndr-brt** [comment]
This will be solved by #58

**Comment by ndr-brt** [comment]
fixed by #90
